bundles/thunk_bundle.js

```diff
@@ -4452,6 +4452,9 @@ var AttributionNodeType;
 init_buffer_shim();
 var import_clone = __toModule(require_clone());
 
+// helpers/object_utils.ts
+init_buffer_shim();
```

Comment:
Interesting, where did this diff come from?

Reply:
Ah, it's just an artifiact of introducing a new import?

Reply:
hmm. I am not sure but I tried to rebuild and the result is not changed. I guess it's just a hard coded stuff from esbuild whenever something is imported.